lighthouse-core/audits/metrics.js

```diff
@@ -37,81 +28,15 @@ class Metrics extends Audit {
-    const totalBlockingTime = await TotalBlockingTime.request(metricComputationData, context); // eslint-disable-line max-len
-
-    /** @type {UberMetricsItem} */
-    const metrics = {
-      // Include the simulated/observed performance metrics
-      firstContentfulPaint: firstContentfulPaint.timing,
-      firstContentfulPaintTs: firstContentfulPaint.timestamp,
-      firstMeaningfulPaint: firstMeaningfulPaint.timing,
-      firstMeaningfulPaintTs: firstMeaningfulPaint.timestamp,
-      largestContentfulPaint: largestContentfulPaint && largestContentfulPaint.timing,
-      largestContentfulPaintTs: largestContentfulPaint && largestContentfulPaint.timestamp,
-      firstCPUIdle: firstCPUIdle && firstCPUIdle.timing,
-      firstCPUIdleTs: firstCPUIdle && firstCPUIdle.timestamp,
-      interactive: interactive && interactive.timing,
-      interactiveTs: interactive && interactive.timestamp,
-      speedIndex: speedIndex && speedIndex.timing,
-      speedIndexTs: speedIndex && speedIndex.timestamp,
-      estimatedInputLatency: estimatedInputLatency.timing,
-      estimatedInputLatencyTs: estimatedInputLatency.timestamp,
-      totalBlockingTime: totalBlockingTime.timing,
-
-      // Include all timestamps of interest from trace of tab
-      observedNavigationStart: traceOfTab.timings.navigationStart,
-      observedNavigationStartTs: traceOfTab.timestamps.navigationStart,
-      observedFirstPaint: traceOfTab.timings.firstPaint,
-      observedFirstPaintTs: traceOfTab.timestamps.firstPaint,
-      observedFirstContentfulPaint: traceOfTab.timings.firstContentfulPaint,
-      observedFirstContentfulPaintTs: traceOfTab.timestamps.firstContentfulPaint,
-      observedFirstMeaningfulPaint: traceOfTab.timings.firstMeaningfulPaint,
-      observedFirstMeaningfulPaintTs: traceOfTab.timestamps.firstMeaningfulPaint,
-      observedLargestContentfulPaint: traceOfTab.timings.largestContentfulPaint,
-      observedLargestContentfulPaintTs: traceOfTab.timestamps.largestContentfulPaint,
-      observedTraceEnd: traceOfTab.timings.traceEnd,
-      observedTraceEndTs: traceOfTab.timestamps.traceEnd,
-      observedLoad: traceOfTab.timings.load,
-      observedLoadTs: traceOfTab.timestamps.load,
-      observedDomContentLoaded: traceOfTab.timings.domContentLoaded,
-      observedDomContentLoadedTs: traceOfTab.timestamps.domContentLoaded,
-
-      // Include some visual metrics from speedline
-      observedFirstVisualChange: speedline.first,
-      observedFirstVisualChangeTs: (speedline.first + speedline.beginning) * 1000,
-      observedLastVisualChange: speedline.complete,
-      observedLastVisualChangeTs: (speedline.complete + speedline.beginning) * 1000,
-      observedSpeedIndex: speedline.speedIndex,
-      observedSpeedIndexTs: (speedline.speedIndex + speedline.beginning) * 1000,
-    };
+    const summary = await TimingSummary.summarize(artifacts, context);
+    /** @type {LH.Artifacts.TimingSummary} */
+    const metrics = summary.metrics;
```

Comment:
seems like we could just destructure these above without the need for type assertions